code/game/objects/items/tanks/jetpack.dm

```diff
@@ -160,6 +160,11 @@
 	distribute_pressure = 0
 	gas_type = /datum/gas/carbon_dioxide
 
+/obj/item/tank/jetpack/carbondioxide/eva
+	desc = "A tank of compressed carbon dioxide for use as propulsion in zero-gravity areas. Painted black to indicate that it should not be used as a source for internals. Rated for less than stellar EVA speeds!"
+	full_speed = FALSE
+	volume = 50
```

Comment:
Why does it have 20 less volume then normal? Even the old tech Void Jetpacks have 60